src/display/canvas.js

```diff
@@ -1004,6 +1004,10 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
       composeSMask(this.ctx, this.current.activeSMask, groupCtx);
       this.ctx.restore();
       copyCtxState(groupCtx, this.ctx);
+
+      if (this.baseTransform) {
+        this.ctx.setTransform.apply(this.ctx, this.baseTransform);
```

Comment:
Can you add a comment that `this.baseTransform` is equal to the groupCtx and current CurrentGraphics combined transform at this point and we need to continue using it instead of restored one from this.ctx?